step-ca.yaml

```diff
@@ -27,13 +27,15 @@ pipeline:
         github.com/go-jose/go-jose/v3@v3.0.4
         golang.org/x/oauth2@v0.27.0
         golang.org/x/crypto@v0.35.0
+        golang.org/x/net@v0.36.0
 
-  - runs: |
-      CGO_ENABLED=0 go build -v \
-        -ldflags='-w -X "main.Version=${{package.version}}"' \
-        -o "${{targets.destdir}}/usr/bin/step-ca" ./cmd/step-ca
-
-  - uses: strip
+  - uses: go/build
+    with:
+      modroot: .
+      packages: ./cmd/step-ca
+      output: step-ca
+      ldflags: |
+        -X main.Version=${{package.version}}
```

Comment:
we can consider including buildtime in here as well. ref: https://github.com/smallstep/certificates/blob/6ce27d3fdbfe2eacc25e48a52267a50579bb93b3/.goreleaser.yml#L42 💭

Reply:
Makes sense, done!